### PR TITLE
[UWP Renderer]: Make references to the ::winrt::AdaptiveCards::Rendering::Uwp namespace consistent

### PR DESCRIPTION
**Pull request by krschau**
# Related Issue

N/A

# Description

Current references to the `::winrt::AdaptiveCards::Rendering::Uwp` namespace are inconsistent. Sometimes it is fully qualified, other times `Rendering::Uwp`, `Uwp`, or most often just `winrt` is used. This PR changes all references to use `winrt::`, except for with `ActionMode`, since there is a type of the same name in the ObjectModel. Here, we have an alias for the namespace, `render_xaml`, that we use in addition.

# How Verified

Manually tested UWP visualizer.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/8094)

### CODE REVIEW

File: source/uwp/Renderer/dll/CppWinRTIncludes.h [hunk 1]
```diff
@@ -54,6 +54,7 @@ namespace winrt
     // In order to avoid "namespace not defined" errors we have to define the namespace here too.
     namespace AdaptiveCards::Rendering::Uwp{}
     using namespace ::winrt::AdaptiveCards::Rendering::Uwp;
+    namespace render_xaml = ::winrt::AdaptiveCards::Rendering::Uwp;
```
Comment (beervoley) [inline source/uwp/Renderer/dll/CppWinRTIncludes.h:57]: I see that in most of the places you simply use `winrt::` namespace to refer to renderer types and you've only declared `render_xaml` to use in cases where types exist in both objectModel and renderer namespaces?
like `enum ActionMode` for example?:) 

just want to make sure that is the intent:)

**Review by beervoley — APPROVED** [review]

**Comment by paulcam206** [comment]
@beervoley - these pipelines should run automatically in the future...

**Pull request merged**